scrapi/base/transformer.py

```diff
@@ -72,8 +72,7 @@ class XMLTransformer(BaseTransformer):
     __metaclass__ = abc.ABCMeta
 
     def _transform_string(self, string, doc):
-        val = doc.xpath(string, namespaces=self.namespaces)
-        return unicode(val[0]) if len(val) == 1 else [unicode(v) for v in val] or ''
+        return doc.xpath(string, namespaces=self.namespaces)
```

Comment:
There were several problems with the removed line, particularly if lists of lxml Elements were returned. I think it is better just to deal with the default xpath return values. I did add the single_result function to help if you expect just a single result.